gatekeeper/manifest.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: "OPA Gatekeeper"
```

Comment:
can you make it 
name: opa-gatekeeper
title: "OPA Gatekeeper"

Reply:
sure, will change it in a few mins and ping you here

Reply:
done @saiyam1814